contributors/devel/automation.md

```diff
@@ -91,24 +83,22 @@ days.
 
 We also run a robotic PR builder that attempts to run tests for each PR.
 
-Before a PR from an unknown user is run, the PR builder bot (`k8s-bot`) asks to
-a message from a contributor that a PR is "ok to test", the contributor replies
-with that message.  ("please" is optional, but remember to treat your robots with
-kindness...)
+Before a PR from an unknown user is run, the PR builder bot (`k8s-ci-robot`) asks to
+a message from a kubernetes member that a PR is `ok-to-test`, the member replies
```

Comment:
"kubernetes member that a PR is safe to test. If it is, the member can reply with the `/ok-to-test` command on a single line to begin CI testing."